packages/editor/src/components/block-inspector/index.js

```diff
@@ -37,13 +37,30 @@ const BlockInspector = ( { selectedBlock, blockType, count } ) => {
 		return <span className="editor-block-inspector__no-blocks">{ __( 'No block selected.' ) }</span>;
 	}
 
+	const hasSupportLink = blockType.support && blockType.support.url && blockType.support.label;
+
 	return (
 		<Fragment>
 			<div className="editor-block-inspector__card">
 				<BlockIcon icon={ blockType.icon } showColors />
 				<div className="editor-block-inspector__card-content">
 					<div className="editor-block-inspector__card-title">{ blockType.title }</div>
 					<div className="editor-block-inspector__card-description">{ blockType.description }</div>
+					{ hasSupportLink && (
+						<div className="editor-block-inspector__card-support">
+							{
+								blockType.support.external ? (
+									<ExternalLink href={ blockType.support.url }>
+										{ blockType.support.label }
```

Comment:
We prolly need to cast this as a string to be on the safe side.

Reply:
> We prolly need to cast this as a string to be on the safe side.

Did you still intend to make this change?